### PR TITLE
Capitalize headers to follow common styling

### PR DESCRIPTION
**Pull request by TLC-10**
"Other useful repositories" --> "Other Useful Repositories"

"Need more?"  --> "Need More?"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Repository | Description
 [YARA](https://github.com/InQuest/awesome-yara)                                     | List of awesome YARA rules, tools, and people
 [Hacker Roadmap](https://github.com/sundowndev/hacker-roadmap)                                     | A guide for amateur pen testers and a collection of hacking tools, resources and references to practice ethical hacking.
 
-## Other useful repositories
```
Comment (Danielferen) [inline README.md:56]: 55

**Review by Danielferen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Benson665) [reply README.md:56]: Ok move on

**Review by SaNthu29 — CHANGES_REQUESTED** [review]
Sedatives LandL

**Comment by cat998** [comment]
已收到！

**Comment by SaNthu29** [comment]
Love virus bit/-^

**Review by Logan0112 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Logan0112) [reply README.md:56]: Muito bem...aprovado...eu tanbem quero

Comment (ravanmaharaj67) [reply README.md:56]: @sri_srillu 
Hack this insta gram account

Comment (deschainej) [inline README.md:56]: - [ ] Hey @Danielferen [
- [ ] ](https://github.com/Hack-with-Github/Awesome-Hacking/pull/138)

**Review by yliasshaloudloween3 — COMMENTED** [review]
55

**Review by Chan9390 — APPROVED** [review]

**Review by Jacknann — COMMENTED** [review]
B

**Pull request merged**